app/assets/javascripts/details/shell/action_bar_bridge.js

```diff
@@ -0,0 +1,116 @@
+var ActionBarBridge;
```

Comment:
This creates a global `ActionBarBridge` variable, doesn't it? Should this instead simply assign `window.ActionBarBridge` to the result of the IIFE?